config/opal_check_pc.m4

```diff
@@ -35,7 +35,7 @@ AC_DEFUN([OPAL_GET_LDFLAGS_FROM_PC], [
   happy=1
   AS_IF([test "$PKG_CONFIG" = ""],
         [happy=0],
-        [OPAL_LOG_COMMAND([pkg_config_results=`$PKG_CONFIG --static --libs-only-L --libs-only-other $1`],
+        [OPAL_LOG_COMMAND([pkg_config_results=`PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/usr/local/lib/pkgconfig $PKG_CONFIG --static --libs-only-L --libs-only-other $1`],
```

Comment:
This may be safe on your system, but is not safe on my system.  If pkg-config doesn't include /usr/local/lib/pkgconfig in the default search path, then users should fix that.  We should not be randomly adding search paths for the user.  Same reason we don't default to searching /usr/local/lib for packages, we only use the default search paths of the compiler.

Reply:
I guess another alternative is to do a hard error when the pkg-config fails instead of printing a warning and continuing. What happened for me was the pkg-config failed to find the installed pmix.pc, and continued on. Later the build will failed to find -lpmix. 

There is a print out suggesting that the users modify their PKG_CONFIG path, but that gets lost in the weeds and is not very user-friendly.

Reply:
You can't hard error when pkg-config fails, because the package config file isn't a required thing to have.  Today, the user-unfriendly stack is as good as it gets, I think.  But I strongly hold to the default search path behavior.  Similar to needing --with-pmix=<path>.